lib/mailer.rb

```diff
@@ -56,6 +56,7 @@ def send(message, attached_file_path = nil)
     params[:charset] = "UTF-8"
     params[:text_part_charset] = "UTF-8"
     if attached_file_path
+      basename = File.basename(attached_file_path).tr("@", "＠")
```

Comment:
__[rubocop]__
* Lint/UselessAssignment: Useless assignment to variable - `basename`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)